server/database/index.js

```diff
@@ -0,0 +1,12 @@
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'ye',
+  database: 'yump_sf_reservation'
+})
+
+connection.connect();
```

Comment:
Definitely doesn't hurt to have this line, but I think it's unnecessary as when you make a query it will implicitly open the connection -- "However, a connection can also be implicitly established by invoking a query:" from https://www.npmjs.com/package/mysql

Reply:
Good observation! Thanks for the feedback.